### DISCUSSION TITLE
Make Polygon use fill="currentColor"

### DISCUSSION THREAD

**Discussion by felquis** [radix-ui/primitives#1226] [Ideas]
On this line:

https://github.com/radix-ui/primitives/blob/4ee9d362fcd5e32edbce9f43f1d41f1bf1a015c1/packages/react/arrow/src/Arrow.tsx#L28

It would be helpful if it uses fill="currentColor" cause I could apply color directly on `<Popover.Arrow className="text-gray-primary" />`

Also, SVG fill=currentColor is very wide spread without @react-icons/all-icons which make it kind of the expected behavior of most SVGs.

What do you think?

**Comment by benoitgrelard** [comment]
Hey @felquis, thanks for raising this.

Whilst it makes sense for icons as they are usually rendered inline along text (color/currentColor), that isn't the case here for the arrow as it doesn't live in a textual context.

That's why I think it makes more sense to style using fill.
With tailwind you can use `fill-blue-500` for that.
